libs/@hashintel/design-system/src/selector-autocomplete.tsx

```diff
@@ -28,7 +28,7 @@ export const TYPE_SELECTOR_HEIGHT = 57;
 export type TypeListSelectorDropdownProps = {
   query: string;
   createButtonProps: Omit<ButtonProps, "children" | "variant" | "size"> | null;
-  variant: "entityType" | "propertyType" | "entity" | "linkType";
+  variant: "entity type" | "property type" | "entity" | "link type";
```

Comment:
I used it like this because these are only used to populate label values. I think it's fine to use kebab case things elsewhere, this is pretty localised